socorro/processor/processor_pipeline.py

```diff
@@ -58,87 +58,32 @@
     UserDataRule,
 )
 
-DEFAULT_RULES = [
-    # fix the raw crash removing null characters
-    DeNullRule,
-    # rules to change the internals of the raw crash
-    ProductRewrite,
-    ESRVersionRewrite,
-    PluginContentURL,
-    PluginUserComment,
-    # rules to transform a raw crash into a processed crash
-    IdentifierRule,
-    MinidumpSha256Rule,
-    BreakpadStackwalkerRule2015,
-    ProductRule,
-    UserDataRule,
-    EnvironmentRule,
-    PluginRule,
-    AddonsRule,
-    DatesAndTimesRule,
-    OutOfMemoryBinaryRule,
-    JavaProcessRule,
-    MozCrashReasonRule,
-    # post processing of the processed crash
-    CrashingThreadRule,
-    CPUInfoRule,
-    OSInfoRule,
-    BetaVersionRule,
-    ExploitablityRule,
-    FlashVersionRule,
-    OSPrettyVersionRule,
-    TopMostFilesRule,
-    ThemePrettyNameRule,
-    MemoryReportExtraction,
-    # a set of classifiers to help with jit crashes
-    JitCrashCategorizeRule,
-    # generate signature now that we've done all the processing it depends on
-    SignatureGeneratorRule,
-]
-
 
 class ProcessorPipeline(RequiredConfig):
-    """this class is a generalization of the Processor into a rule processing
-    framework. This class is suitable for use in the 'processor_app'
-    introducted in 2012."""
+    """Processor pipeline for Mozilla crash ingestion."""
 
     required_config = Namespace('transform_rules')
-    required_config.add_option(
+
+    # BreakpadStackwalkerRule2015 configuration
+    required_config.breakpad = Namespace()
+    required_config.breakpad.add_option(
         'dump_field',
         doc='the default name of a dump',
         default='upload_file_minidump',
     )
-    required_config.add_option(
+    required_config.breakpad.add_option(
         'command_pathname',
         doc='the full pathname to the external program to run (quote path with embedded spaces)',
-        # NOTE(willkg): This is the path for the RPM-based Socorro deploy. When
-        # we switch to Docker, we should change this.
-        default='/data/socorro/stackwalk/bin/stackwalker',
-    )
-    required_config.add_option(
-        'result_key',
-        doc=(
-            'the key where the external process result should be stored '
-            'in the processed crash'
-        ),
-        default='stackwalker_result',
-    )
-    required_config.add_option(
-        'return_code_key',
-        doc=(
-            'the key where the external process return code should be stored '
-            'in the processed crash'
-        ),
-        default='stackwalker_return_code',
+        default='/stackwalk/stackwalker',
```

Comment:
Since we're using Docker across the board now, we can set this to the path in the Docker container and remove the note I added many moons ago.